style.css

```diff
@@ -25,4 +25,18 @@
 .dokuwiki q.blockquote-plugin {
     border: 1px dotted __border__;
     background: __background_alt__;
+    quotes: "\0022" "\0022" "\0027" "\0027";
+    /*
+" 	double quote 	\0022
```

Comment:
Would you mind removing this large comment block? Reminders are nice but this one's too large in my opinion.